packages/app/src/explorer/utils.ts

```diff
@@ -74,3 +74,17 @@ export function focusPrevious(e: KeyboardEvent<HTMLButtonElement>) {
     e.preventDefault();
   }
 }
+
+export function focusFirst(e: KeyboardEvent) {
+  const buttonList = getExplorerButtonList();
+
+  buttonList[0]?.focus();
+  e.preventDefault();
+}
+
+export function focusLast(e: KeyboardEvent) {
+  const buttonList = getExplorerButtonList();
+
+  buttonList[buttonList.length - 1]?.focus();
+  e.preventDefault();
+}
```

Comment:
As `focusFirst` and `focusLast` are called in the `Explorer`, I could have used `ref.current` but I preferred to follow the other `focus..` method template.